pytest-pixl/src/pytest_pixl/dicom.py

```diff
@@ -14,308 +14,114 @@
 
 """Functions to write test DICOM files."""
 
+from __future__ import annotations
+
 import importlib
 import json
 
 import numpy as np
-from pydicom.dataset import Dataset, FileMetaDataset
-from pydicom.sequence import Sequence
+from pydicom.datadict import dictionary_has_tag
+from pydicom.dataset import Dataset
 
 
 def write_volume(filename_pattern: str):
```

Comment:
Missing return type annotation (`-> None`)

Reply:
Nice, clearly another indication that `mypy` is not working properly in the CI (#305) 🫠